tardis/io/schemas/montecarlo.yml

```diff
@@ -17,7 +17,7 @@ properties:
   no_of_packets:
     type: number
     multipleOf: 1.0
-    description: Seed for the random number generator
+    description: The number of packets used in each iteration depends on the specific simulation software and the input parameters provided by the user.
```

Comment:
This is not the correct description. It should simply read "The number of packets used in each iteration".